### PR TITLE
User interface with React and antd

### PR DESCRIPTION
**Pull request by bsekachev**
- GitHub link in the header
- Added bug tracker link
- Added dumpers and loaders with nice indication
- Different bug fixes and UI improvements
- Removed prebuilt cvat-core
- Added favicon.ico
- Great refactoring :) 

@denStrigo, could you please look at the PR?

### CODE REVIEW

File: cvat-ui/src/actions/formats-actions.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import getCore from '../core';
+
+const cvat = getCore();
+
+export enum FormatsActionTypes {
+    GETTING_FORMATS_SUCCESS = 'GETTING_FORMATS_SUCCESS',
+    GETTING_FORMATS_FAILED = 'GETTING_FORMATS_FAILED',
+}
+
+export function gettingFormatsSuccess(formats: any): AnyAction {
```
Comment (nmanovic) [inline cvat-ui/src/actions/formats-actions.ts]: I would prefer to have names like "onSuccessServerFormats" and "OnFailureServerFormats".

File: cvat-ui/src/actions/formats-actions.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+export function gettingFormatsFailed(gettingFormatsError: any): AnyAction {
+    return {
+        type: FormatsActionTypes.GETTING_FORMATS_FAILED,
+        payload: {
+            gettingFormatsError,
+        },
+    };
+}
+
+export function gettingFormatsAsync(): ThunkAction<Promise<void>, {}, {}, AnyAction> {
```
Comment (nmanovic) [inline cvat-ui/src/actions/formats-actions.ts]: getServerFormatsAsync

File: cvat-ui/src/actions/tasks-actions.ts [hunk 3]
```diff
@@ -7,42 +7,60 @@ import getCore from '../core';
...
+function getTasksSuccess(array: any[], previews: string[],
+    count: number, gettingQuery: TasksQuery): AnyAction {
     const action = {
         type: TasksActionTypes.GET_TASKS_SUCCESS,
         payload: {
             previews,
             array,
             count,
-            query,
+            gettingQuery,
```
Comment (nmanovic) [inline cvat-ui/src/actions/tasks-actions.ts]: I don't think that 'gettingQuery' is better than 'query'.

File: cvat-ui/src/actions/tasks-actions.ts [hunk 4]
```diff
@@ -7,42 +7,60 @@ import getCore from '../core';
...
 }
 
-export function getTasksFailed(error: any, query: TasksQuery): AnyAction {
+function getTasksFailed(tasksFetchingError: any, gettingQuery: TasksQuery): AnyAction {
     const action = {
         type: TasksActionTypes.GET_TASKS_FAILED,
         payload: {
-            error,
-            query,
+            tasksFetchingError,
```
Comment (nmanovic) [inline cvat-ui/src/actions/tasks-actions.ts]: Again, why do you need to rename these variables? Names were perfect, short, and meaningful.

File: cvat-ui/src/actions/formats-actions.ts (hunk 1 above)
Comment (bsekachev) [reply cvat-ui/src/actions/formats-actions.ts]: I believe ServerFormats isn't the best choice.
AnnotationsFormats or shorter variant just Formats.

File: cvat-ui/src/actions/tasks-actions.ts (hunk 3 above)
Comment (bsekachev) [reply cvat-ui/src/actions/tasks-actions.ts]: Returned this name here

File: cvat-ui/src/actions/tasks-actions.ts (hunk 4 above)
Comment (bsekachev) [reply cvat-ui/src/actions/tasks-actions.ts]: Returned this name here

**Pull request merged**